log4j2-ecs-layout/src/test/java/co/elastic/logging/log4j2/EcsLayoutWithStackTraceAsArrayTest.java

```diff
@@ -0,0 +1,45 @@
+package co.elastic.logging.log4j2;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class EcsLayoutWithStackTraceAsArrayTest extends Log4j2EcsLayoutTest {
+
+    @Override
+    protected EcsLayout.Builder configureLayout(LoggerContext context) {
+        return super.configureLayout(context)
+                .setExceptionPattern("%cEx")
```

Comment:
Why use the custom one and not rely these tests on the default pattern converter that is already configured through the superclass? Unless there's a good reason, I prefer to here what most users will use by default.

Reply:
I can do that, I was using the custom converter to keep the stack traces predictable as I don't really want to be testing whatever log4j2 does under the covers (beyond confirming that it works anyway).  I'll split the custom converter to its own test and use the built-in ones for these tests.

Reply:
It's already tested through `EcsLayoutWithExceptionPatternTest`, so what do you mean split further?
I understand the need for consistency without relying on log4j internals, but maybe you can use the builtin converter to achieve exactly that - filter out stack trace elements that are not yours and limit to only 3. Can you do that?

Reply:
> It's already tested through `EcsLayoutWithExceptionPatternTest`, so what do you mean split further?

You're right, I did, too much multitasking this week.  😅 

> I understand the need for consistency without relying on log4j internals, but maybe you can use the builtin converter to achieve exactly that - filter out stack trace elements that are not yours and limit to only 3. Can you do that?

Definitely, I'll try to get a PR up shortly.